admin_guide/advanced_scheduling/pod_affinity.adoc

```diff
@@ -0,0 +1,351 @@
+    image: gcr.io/google_containers/pause:2.0
+
+----
+====
+<1> Stanza to configure pod anti-affinity.
+<2> Defines a preferred rule.
+<3> Specifies a weight for a preferred rule. The node that with highest weight is preferred.
+<4> The key and value (label) that must be matched to apply the rule.
+<5> The xref:../../rest_api/openshift_v1.html#unversioned-labelselectorrequirement[operator] represents the relationship between the key and a set of values.  Can be `In`, `NotIn`, `Exists`, or `DoesNotExist`.
+
+
+[NOTE]
+====
+If labels on a node change at runtime such that the affinity rules on a pod are no longer met, the pod will still continue to run on the node.
+====
+
+[[admin-guide-sched-affinity-config-pod-req]]
+=== Configuring an Affinity Rule
+
+The following steps demonstrate a simple two-pod configuration that creates pod with a label and a pod that uses affinity to allow scheduling with that pod.
+
+. Create a pod with a specific label in the pod specification:
++
+----
+$ cat team4.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: team4
+  labels:
+     team: "4"
+spec:
+  containers:
+  - name: ocp
+    image: docker.io/ocpqe/hello-pod
+----
+
+. When creating other pods, edit the pod specification to set the following parameters:
+
+. Use the `podAffinity` stanza to configure the `requiredDuringSchedulingIgnoredDuringExecution` parameter or `preferredDuringSchedulingIgnoredDuringExecution` parameter:
++
+..Specify the key and value that must be met. If you want the new pod to be scheduled with the other pod, use the same `key` and `value` parameters as the label in the first pod.
++
+----
+    podAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: security
+            operator: In
+            values:
+            - S1
+        topologyKey: failure-domain.beta.kubernetes.io/zone
+----
++
+.. Specify an `operator`. The xref:../../rest_api/openshift_v1.html#unversioned-labelselectorrequirement[operator can be `In`, `NotIn`, `Exists`, or `DoesNotExist`]. For example, use the operator `In` to require the label to be in the node.
++
+.. Specify a `topologyKey`, which is a prepopulated link:https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels[Kubernetes label] that the system uses to denote such a topology domain.
+
+. Create the pod.
++
+----
+$ oc create -f <pod-spec>.yaml
+----
+
+[[admin-guide-sched-affinity-config-pod-pref]]
+=== Configuring an Anti-affinity Rule
+
+The following steps demonstrate a simple two-pod configuration that creates pod with a label and a pod that uses anti-affinity to prevent scheduling with that pod.
+
+. Create a pod with a specific label in the pod specification:
++
+----
+$ cat team4.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: team4
+  labels:
+     team: "4"
+spec:
+  containers:
+  - name: ocp
+    image: docker.io/ocpqe/hello-pod
+----
```

Comment:
for matching step 3, you might want use this to replace team4.yaml
cat pod-s2.yaml
```
apiVersion: v1
kind: Pod
metadata:
  name: security-s2
  labels:
    security: S2
spec:
  containers:
  - name: security-s2
    image: docker.io/ocpqe/hello-pod
```